src/django_components/component_media.py

```diff
@@ -865,9 +865,14 @@ def resolve_media_file(
 
     filepath_abs_or_glob = os.path.join(media_root_dir, filepath)
 
+    # If the path is a URL, don't resolve it
+    # (e.g. https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.0.2/chart.min.js)
+    # This is defined based on Django's `Media.absolute_path()` method.
+    is_url_path = filepath.startswith(("http://", "https", "/"))
```

Comment:
The fix is simply to check if a path is URL, and skip the glob resolution if so.